src/libpriv/rpmostree-core.c

```diff
@@ -1933,6 +2004,7 @@ rpmostree_context_assemble_commit (RpmOstreeContext      *self,
                                    int                    tmprootfs_dfd,
                                    OstreeRepoDevInoCache *devino_cache,
                                    const char            *parent,
+                                   gboolean               noscripts,
                                    char                 **out_commit,
                                    GCancellable          *cancellable,
                                    GError               **error)
```

Comment:
I guess it's inevitable that the param list keeps growing here since this thing does so many things now. We should probably in the future split it into smaller ones before it becomes a monster. For `noscripts` specifically, it seems like it could just be something we  put in `RpmOstreeContext` (maybe it's time for a `flags` member there). But we can leave it as is for now until we do the splitting.